lib/v2/xinhua-daily-telegraph/daily.js

```diff
@@ -0,0 +1,71 @@
+const timezone = require('@/utils/timezone');
+const cheerio = require('cheerio');
+const { parseDate, getElementChildrenInnerText, getHTMLWithCache } = require('./utils');
+
+module.exports = async (ctx) => {
+    const date = timezone(new Date(), 8);
+    const dateFormatted = date.getFullYear().toString() + ('0' + (date.getMonth() + 1).toString()).slice(-2) + date.getDate().toString();
+
+    const baseUrl = `http://mrdx.cn/content/${dateFormatted}`;
+    const link = `${baseUrl}/Page01DK.htm`;
+
+    const $ = cheerio.load(await getHTMLWithCache(link, ctx));
+
+    const passagesList = $('.listdaohang > ul > li a');
+    const passageLinks = [];
+
+    // 当日所有文章Url在<a>的`daoxiang` attribute中
+    // 其站点使用**非标准**的HTML语法
+    // eg: `<a class="foo" daoxiang="foo-bar.htm">Lorem ipsum</a>`
+    for (let i = 0; i < passagesList.length; i++) {
+        passageLinks.push(passagesList[i].attribs.daoxiang);
+    }
+
+    const passages = [];
+    const results = await Promise.all(
+        passageLinks.map(async (passage) => {
+            const link = `${baseUrl}/${passage}`;
+            const response = await getHTMLWithCache(link, ctx);
```

Comment:
Caching the HTML response and letting RSSHub process it again every time someone fetches the feed may not be a good idea. Instead, we can cache the processed result and return it.